src/components/Header.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link, NavLink } from 'react-router-dom';
+import log from '../images/logo.png';
+
+const Header = () => (
+  <div className="header-container">
+    <div className="header-item">
+      <Link className="nav-link mt-1" to="/">
+        <img className="logo" src={log} alt="logo" />
+        <span className="logo-text"> Space Travelers&apos; Hub </span>
+      </Link>
+    </div>
+    <div className="header-item">
+      <ul className="nav justify-content-end">
+        <li className="nav-item">
+          <NavLink className="nav-link" to="/"> Rockets</NavLink>
+        </li>
+        <li className="nav-item">
+          <NavLink className="nav-link" to="/missions">Missions</NavLink>
+        </li>
+        <li className="nav-item">
+          <NavLink className="nav-link" to="/my-profile">My Profile</NavLink>
+        </li>
+      </ul>
+    </div>
+  </div>
+);
+
+export default Header;
```

Comment:
- [ ] Pages are reloading when changing from one view to another. When I reserved rockets and joined missions and then click on the My Profile link, when I navigate back to rockets and mission pages, these rockets are no longer reserved. Data should not be added to the store on every re-render.

[Screencast from 08-12-2023 04:22:33 PM.webm](https://github.com/Grandi0z/Space_travelers_hub/assets/63087491/341bf9c5-6751-42c0-9b26-eb723dc34d22)